Security-Team.md

```diff
@@ -142,6 +142,9 @@ The expected workflow for issues reported to Node.js is:
      getting resolved. Comments on the issue when someone is working on it are
      useful, as is assignment of the issue to a person who agrees to be
      responsible for following it through to resolution and disclosure.
+   - Once a fix is ready for release, the issue should be assigned to the group
+     `Ready for release`, and a comment added indicating the relevant PRs that
+     need to be merged during the release process.
```

Comment:
It's not clear to me what "assigned to the group Ready for release" means. Is there maybe a missing word or something?

Reply:
If that's a HackerOne team, then yes.

Reply:
Perhaps the team could simply  be called `Releasers`?